cpp/include/cudf/strings/repeat_strings.hpp

```diff
@@ -18,6 +18,8 @@
 #include <cudf/scalar/scalar.hpp>
 #include <cudf/strings/strings_column_view.hpp>
 
+#include <optional>
```

Comment:
The `<optional>` include is being addressed in #10044

Reply:
Reverted.